game/static/game/css/lesson.css

```
@@ -371,3 +371,19 @@ h1 {
     pointer-events: none;
     transition: opacity 0.2s ease;
 }
+/*KEYBOARD NAVIGATION */
```

Stylelint (17.13.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)